The building system of chromium on Termux is much complex. For the host
toolchain, it uses clang-15 from Ubuntu as CXX compiler and libstdcxx-10
from Google-provided sysroot as CXX headers. For the target toolchain, it
uses CXX compiler and headers from Android NDK, which is from clang-17 and
libcxx-17. This patch is to satisfy them all.

--- a/components/autofill/core/browser/data_model/autofill_i18n_api.h
+++ b/components/autofill/core/browser/data_model/autofill_i18n_api.h
@@ -15,8 +15,8 @@
 // Country code that represents autofill's legacy address hierarchy model as
 // stored `kAutofillModelRules`. As a workaround for GCC we declare the
 // std::string constexpr first.
-constexpr inline std::string kLegacyHierarchyCountryCodeString{"XX"};
-constexpr AddressCountryCode kLegacyHierarchyCountryCode =
+constexpr inline const char kLegacyHierarchyCountryCodeString[] = "XX";
+static const inline AddressCountryCode kLegacyHierarchyCountryCode =
     AddressCountryCode(kLegacyHierarchyCountryCodeString);
 
 // Creates an instance of the address hierarchy model corresponding to the
